ztp/policygenerator/policyGen/policyBuilderOverlay_test.go

```diff
@@ -103,6 +117,13 @@ spec:
   sourceFiles:
     - fileName: GenericCR.yaml
       policyName: "gen-policy1"
+      metadata:
+        labels:
+          key1: newvalue
+          key3: value3
```

Comment:
Use key/values for labels that are different from annotations to catch any bugs where the code accidentally uses the wrong datastructure.

Reply:
Updated